charts/geth-prysm/templates/geth.configmap.yaml

```diff
@@ -0,0 +1,122 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: geth-config
+data:
+  password.txt:
+  node1key: |-
+    {"address":"123463a4b065722e99115d6c222f267d9cabb524","crypto":{"cipher":"aes-128-ctr","ciphertext":"93b90389b855889b9f91c89fd15b9bd2ae95b06fe8e2314009fc88859fc6fde9","cipherparams":{"iv":"9dc2eff7967505f0e6a40264d1511742"},"kdf":"scrypt","kdfparams":{"dklen":32,"n":262144,"p":1,"r":8,"salt":"c07503bb1b66083c37527cd8f06f8c7c1443d4c724767f625743bd47ae6179a4"},"mac":"6d359be5d6c432d5bbb859484009a4bf1bd71b76e89420c380bd0593ce25a817"},"id":"622df904-0bb1-4236-b254-f1b8dfdff1ec","version":3}
+  genesis.json: |-
+    {
+      "config": {
+        "chainId": {{ .Values.general.networkId }},
+        "homesteadBlock": 0,
+        "daoForkSupport": true,
+        "eip150Block": 0,
+        "eip155Block": 0,
+        "eip158Block": 0,
+        "byzantiumBlock": 0,
+        "constantinopleBlock": 0,
+        "petersburgBlock": 0,
+        "istanbulBlock": 0,
+        "muirGlacierBlock": 0,
+        "berlinBlock": 0,
+        "londonBlock": 0,
+        "arrowGlacierBlock": 0,
+        "grayGlacierBlock": 0,
+        "shanghaiTime": 1699271757,
+        "terminalTotalDifficulty": 0,
+        "terminalTotalDifficultyPassed": true
+      },
+      "nonce": "0x0",
+      "timestamp": "0x6548d44d",
+      "extraData": "0x0000000000000000000000000000000000000000000000000000000000000000123463a4b065722e99115d6c222f267d9cabb5240000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+      "gasLimit": "0x1c9c380",
+      "difficulty": "0x1",
+      "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+      "coinbase": "0x0000000000000000000000000000000000000000",
+      "alloc": {
```

Comment:
might need more accounts here. If we can add accounts with balance later that will work as well. 
Use Case - There are tests where we use close to 20 networks in one test. In that case for parallel contract deployments across those multiple networks if we use similar owner account the contract address becomes same for multiple networks. CL node cannot support same contract address for different networks yet. We might need around 20 accounts here so that the test can use diff owner accounts for diff networks.

Reply:
yes, I can do that the same way it's done for besu :-)